pkg/operator/controllers/guardrails/policies/README.md

```diff
@@ -31,54 +31,53 @@ Constraint is manually created
 apiVersion: templates.gatekeeper.sh/v1
 kind: ConstraintTemplate
 metadata:
-  name: aroprivilegednamespace
+  name: arodenyprivilegednamespace
```

Comment:
I don't have a lot of context for most of this, but what's the reason for adding deny to the string, and how confusing will that make any other parts of the system/onboarding this change? I assume this naming clarifies functionality, but changing names can be confusing. Is the change documented?

Reply:
the story is a bit long, originally this policy does not have a deny in its name, and while more and more policies added, all others have arodeny prefix to imply this is a deny policy, and we decided to use that as sort of uniforms, hence is the change.
as to the confusion, the policy is not published yet, this is sort of last chance to unify our namings, the customer facing doc has adapted to the change IIRC.

Reply:
That makes sense. Thanks.